-EnvironmentName to -Environment  using PS 1.2.11

JakobGSvendsen opened the pull request:
it is in the migration guide that this parameter changes name

https://github.com/Azure/azure-powershell/blob/preview/documentation/release-notes/Stack/migration-guide.1.2.11.md

azurecla commented:
Hi __@v-anpasi__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

JakobGSvendsen committed:
-EnvironmentName to -Environment  using PS 1.2.11

it is in the migration guide that this parameter changes name

https://github.com/Azure/azure-powershell/blob/preview/documentation/release-notes/Stack/migration-guide.1.2.11.md
